company-profiles/cancom.md

```diff
@@ -0,0 +1,32 @@
+# CANCOM
+
+## Company blurb
+
+As a Leading Digital Transformation Partner, CANCOM is not only a reliable IT service provider for its customers, but rather the holistic business partner for companies, organizations and the public sector. CANCOM turns IT into the future!
+
+## Company size
+
+It has 4,000+ employees with offices across europe.
+
+## Remote status
+
+CANCOM is currently remote-friendly, and flexible.
+
+## Region
+
+Germany/Europe
```

Comment:
The region here and in the readme file are for where the company hires. 

Is CANCOM only hiring in Germany or "across Europe" as mentioned in `Company size` ?

Reply:
@dougaitken thanks for your comment.
`Company size` part is where the offices are located and people are working from. It doesn't necessary mean that these offices/locations are _actively_ hiring. Or maybe I am missing something here.
Nevertheless, we are actively hiring in Germany, Austria and Slovakia. However, I am open to hiring people from anywhere in Europe, although not clearly stated on the website.

Do you want me to adjust Region in `company-profile` and `readme` to `Germany/Austria/Slovkia` for now?

Reply:
Let's go for where you're actively hiring please @Seenox
Once this is merged, you can update whenever you'd like.

Reply:
@dougaitken Alright! I've pushed the changes. Please review (and adjust again if necessary by yourself) and merge.
Thanks!